mavros/src/lib/ftf_frame_conversions.cpp

```diff
@@ -85,21 +107,61 @@ Covariance3d transform_static_frame(const Covariance3d &cov, const StaticTF tran
 	}
 }
 
-#if 0
 Covariance6d transform_static_frame(const Covariance6d &cov, const StaticTF transform)
 {
-	//! @todo implement me!!!
+	Covariance6d cov_out_;
+	EigenMapConstCovariance6d cov_in(cov.data());
+	EigenMapCovariance6d cov_out(cov_out_.data());
+
+	Eigen::MatrixXd affine6dTF(6, 6);
```

Comment:
Why?! MatrixXd is **dynamic**! Since you know dimensions there no reason not to use **static** variation by passing size to Matrix template.

```cpp
using Matrix6d = Eigen::Matrix<double, 6, 6>;
using Matrix9d = Eigen::Matrix<double, 9, 9>;
```